cli/args/mod.rs

```diff
@@ -1355,7 +1355,7 @@ impl CliOptions {
     } else if self.maybe_package_json.is_some() {
       Ok(Default::default())
     } else {
-      bail!("No config file found")
+      bail!("deno task error: No config file found. See https://deno.land/manual@v{}/getting_started/configuration_file", env!("CARGO_PKG_VERSION"))
```

Comment:
nit
```suggestion
      bail!("deno task couldn't find deno.json(c). See https://deno.land/manual@v{}/getting_started/configuration_file", env!("CARGO_PKG_VERSION"))
```
The current change prints error twice and "config file" is a generic term in the context of a multi-lang environment:
```
error: deno task error: No config file found. See https://deno.land/manual@v1.43.6/getting_started/configuration_file
``` 

The suggestion improves on it by explicitly stating that `deno.json` or `deno.jsonc` is not found and removes the "error" duplication.
```
error: deno task couldn't find deno.json(c). See https://deno.land/manual@v1.43.6/getting_started/configuration_file
```